CHANGELOG.md

```diff
@@ -4,6 +4,260 @@ This changelog goes through all the changes that have been made in each release
+  * Migrate from Macaron to Chi framework (#14293)
+  * Deprecate building for mips (#14174)
+  * Consolidate Logos and update README header (#14136)
+  * Inline manifest.json (#14038)
+  * Store repository data in data path if not previously set (#13991)
+  * Rename "gitea" png to "logo" (#13974)
+  * Standardise logging of failed authentication attempts in internal SSH (#13962)
+  * Add markdown support in organization description (#13549)
+  * Improve users management through the CLI (#6001) (#10492)
+* FEATURES
+  * Repository transfer has to be confirmed, if user can not create repo for new owner (#14792)
+  * Allow blocking some email domains from registering an account (#14667)
+  * Create a new issue based on reference to an issue comment (#14366)
+  * Add support to migrate from gogs (#14342)
+  * Add pager to the branches page (#14202)
+  * Minimal OpenID Connect implementation (#14139)
+  * Display current stopwatch in navbar (#14122)
+  * Display SVG files as images instead of text (#14101)
+  * Disable SSH key deletion of externally managed Keys (#13985)
+  * Add support for ed25519_sk and ecdsa_sk SSH keys (#13462)
+  * Add support for Mastodon OAuth2 provider (#13293)
+  * Add gitea sendmail command (#13079)
+  * Create DB session provider(based on xorm) (#13031)
+  * Add dismiss review feature (#12674)
+  * Make manual merge autodetection optional and add manual merge as merge method (#12543)
+  * Dump github/gitlab/gitea repository data to a local directory and restore to gitea (#12244)
+  * Create Rootless Docker image (#10154)
+* API
+  * Get pull, return head branch sha, even if deleted (#14931)
+  * Export LFS & TimeTracking function status (#14753)
+  * Show Gitea version in swagger (#14654)
+  * Fix PATCH /repos/{owner}/{repo} panic (#14637)
+  * Add Restricted Field to User (#14630)
+  * Add support for ref parameter to get raw file API (#14602)
+  * Add affected files of commits to commit struct (#14579)
+  * Add delete release by tag & delete tag (#14563) & (#13358)
+  * Add pagination to ListBranches (#14524)
+  * Add signoff option in commit form (#14516)
+  * GetRelease by tag only return release (#14397)
+  * Add MirrorInterval to the API (#14163)
+  * Make BasicAuth Prefix case insensitive (#14106)
+  * Add user filter to issueTrackedTimes, enable usage for issue managers (#14081)
+  * Add ref to create/edit issue options & deprecated assignee (#13992)
+  * Add Ref to Issue (#13946)
+  * Expose default theme in meta and API (#13809)
+  * Send error message when CSRF token is missing (#13676)
+  * List, Check, Add & delete endpoints for repository teams (#13630)
+  * Admin EditUser: Make FullName, Email, Website & Location optional (#13562)
+  * Add more filters to issues search (#13514)
+  * Add review request api (#11355)
+* BUGFIXES
+  * Ensure validation occurs on clone addresses too (#14994)
+  * Fix several render issues (#14986)
```

Comment:
is SECURITY - title can be changed to "Fix several issues highlighted during fuzzing"